src/pages/cli/graphql/data-modeling.mdx

```diff
@@ -132,6 +132,37 @@ query MyQuery {
 }
 ```
 
+Beginning in Amplify CLI version 8.5.0, there is a feature flag named `enableautoindexquerynames`. If enabled, this flag allows configuration of secondary indexes without any explicit naming.
```

Comment:
We don't usually put CLI version information into the "guides" section of the docs. The right place for this is in: https://docs.amplify.aws/cli/migration/* path. Also isn't the feature flag enabled by default for new apps? For customers without the feature flag, we should have the client print a warning rather to explain the gap in the FF.

Reply:
That's correct, it will be once we merge in the related CLI PR. I'll open an AI to add the necessary validation check and warn in the transformer. Instead of adding a migration guide.